easyInterface/Diffraction/Interface.py

```diff
@@ -473,7 +507,7 @@ def setPhase(self, phase: Phase) -> NoReturn:
         if isinstance(phase, Phase):
             new_phase_name = phase['phasename']
             if new_phase_name in self.project_dict['phases'].keys():
-                k, v = self.project_dict.getItemByPath(['phases', new_phase_name]).dictComparison(phase)
+                k, v, t = self.project_dict.getItemByPath(['phases', new_phase_name]).dictComparison(phase)
```

Comment:
Should we use a less cryptic names, such as `keys`, `values`? And what is `t`? It seems that `t` is not used.

Reply:
Modified